### PR TITLE
Update to match existing instances.

### PR DESCRIPTION
**Pull request by jmcarp**
I took a look at the existing FEC database configuration and started to update the terraform configuration to match. @ccostino: can you review and comment / update if there's more that should get changed?

### CODE REVIEW

File: terraform/fec_rds.tf [hunk 1]
```diff
@@ -73,10 +70,10 @@ resource "aws_db_instance" "rds_production" {
   }
   engine = "postgres"
   engine_version = "9.6.1"
```
Comment (ccostino) [inline terraform/fec_rds.tf:72]: I think I called this out before, but we're definitely excited to use this as an opportunity to also upgrade to 9.6.1.  Thank you!

File: terraform/fec_rds.tf [hunk 2]
```diff
@@ -73,10 +70,10 @@ resource "aws_db_instance" "rds_production" {
...
-  instance_class = "db.t2.micro"
-  allocated_storage = 10
-  name = "fec_production"
-  username = "${var.rds_production_username}"
+  instance_class = "db.r3.2xlarge"
+  allocated_storage = 1650
```
Comment (ccostino) [inline terraform/fec_rds.tf]: I just spoke with Rohan and Wei at FEC, we'd like to bump this up to 2 TB (`2000`) for all environments.

File: terraform/fec_rds.tf [hunk 3]
```diff
@@ -103,10 +100,10 @@ resource "aws_db_instance" "rds_staging" {
   }
   engine = "postgres"
   engine_version = "9.6.1"
-  instance_class = "db.t2.micro"
-  allocated_storage = 10
-  name = "fec_staging"
-  username = "${var.rds_staging_username}"
+  instance_class = "db.r3.2xlarge"
+  allocated_storage = 1350
```
Comment (ccostino) [inline terraform/fec_rds.tf]: Same note as above about bumping size up (`2000`).

File: terraform/fec_rds.tf [hunk 4]
```diff
@@ -123,10 +120,10 @@ resource "aws_db_instance" "rds_development" {
   }
   engine = "postgres"
   engine_version = "9.6.1"
-  instance_class = "db.t2.micro"
-  allocated_storage = 10
-  name = "fec_development"
-  username = "${var.rds_development_username}"
+  instance_class = "db.r3.2xlarge"
+  allocated_storage = 1050
```
Comment (ccostino) [inline terraform/fec_rds.tf]: Same note as above about bumping size up (`2000`).

**Review by ccostino — COMMENTED** [review]
Thank you, @jmcarp!  I made a couple of notes about the size changes we'd also like to see (confirmed from Wei and Rohan at FEC), but I also wanted to call out that we have the replica in `dev` as well, and I don't see that in here.

Also, I know the production database (and its replica) have provisioned IOPS as opposed to the general SSD setting.  Does that also need to be captured here?

Thanks again for your help with this, it is greatly appreciated!

**Comment by jmcarp** [comment]
@ccostino: do you have access to update this branch? If not, can you ask @admins-github to set that up? I don't have admin anymore, or I'd check myself.

**Comment by ccostino** [comment]
Will do; I tried to login myself earlier to check and couldn't get in, so I have to have my account credentials reset/re-enabled as well.

**Comment by ccostino** [comment]
^^ meant that I also tried to verify in the AWS account this info currently lives in for the current instances but couldn't get in.  For the repo, I'll give this a shot and if not, reach out about that too.

**Comment by ccostino** [comment]
I just tried, I don't have access either; I asked in Slack to get help with this.

**Comment by ccostino** [comment]
@jmcarp: we should be set with access here now (looks like I am based on what I see).

**Comment by jmcarp** [comment]
@ccostino: updated. Can you merge or update if there's more to do here?

**Comment by ccostino** [comment]
We were still tied up in on-site PI planning at FEC yesterday; I just did a quick look though and everything looks fine to me, thanks again @jmcarp!

**Pull request merged**